### PR TITLE
tank resprite

### PR DESCRIPTION
**Pull request by tyeagg**
## About The Pull Request
resprites tank also renames it because couldnt bother to resprite the logo
image

https://github.com/tgstation/TerraGov-Marine-Corps/assets/98577688/58175951-be9d-48a0-b271-133cb5afaada

## Why It's Good For The Game
better looking and scifi setting
## Changelog
:cl:
imageadd: new tank module sprites
imageadd: new tank weapon sprites
imageadd: new tank sprite
/:cl:

### CODE REVIEW

**Comment by tyeagg** [comment]
forgot to update damage sprites

**Comment by tyeagg** [comment]
dont merge noticed a mistake

**Comment by Pockets-byte** [comment]
this results in environment camo being removed

**Review by Wirpy — APPROVED** [review]
sprite approval, merge that fat boy

File: code/modules/vehicles/armored/armored_modules.dm [hunk 1]
```diff
@@ -14,6 +14,8 @@
 	var/obj/vehicle/sealed/armored/owner
 	///Bool whether this module is a driver module or not
 	var/is_driver_module = TRUE
+	///Bool whether this module is attached to the hull or not
+	var/is_hull_module = FALSE
```
Comment (TiviPlus) [inline code/modules/vehicles/armored/armored_modules.dm:18]: clarify it's for visuals pls

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Don't see zoom in the icon diff?
Also APC sprites look like they've been shifted down, make sure it looks fine if you havent already
Missing the tread animation?

**Comment by tyeagg** [comment]
> Don't see zoom module in the icon diff?
> Also APC sprites look like they've been shifted down, make sure it looks fine if you havent already
> Missing the tread animation?

Zoom module is just very small its the red eyes on the turret

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**